src/components/CounterBadge/content-validation.js

```diff
@@ -0,0 +1,17 @@
+export const textOfMaxTwoSymbols = (props, propName, componentName = 'CounterBadge') => {
```

Comment:
I just thought about it, and in the future we will have typescript, what's then?
Maybe we would like to do this check in the constructor, and then to throw in case length > 2?